src/components/form/__definition__.js

```diff
@@ -45,6 +46,7 @@ let definition = new Definition('form', Form, {
     summary: "Boolean",
   },
   propDescriptions: {
+    autoDisable: "Set to true will disable the save button on submit",
```

Comment:
Please change this to: "Set to true to disable the save button once the form has been submitted".